src/components/css/AboutUsPage.css

```diff
@@ -3,31 +3,28 @@
     max-Height: 100vh;
 }
 
-h1 {
+#mainTitle {
+    font-family: "Calibri Light";
+    font-size: 50px;
     text-align: center;
-    font-family: Calibri;
 }
 
-#title {
-    text-align: center;
+#smallTitle {
     font-family: Calibri;
-    text-decoration: underline #ffffff;
-    text-shadow: 2px 2px 5px dodgerblue;
-    font-size: 22px;
+    font-size: 25px;
+    text-align: left;
 }
 
 #version {
-    text-align: center;
     font-family: Calibri;
-    text-decoration: underline dodgerblue;
-    text-shadow: 2px 2px 5px mediumspringgreen;
     font-size: 16px;
+    text-align: left;
 }
 
 p {
-    text-align: center;
     font-family: Calibri;
-    margin-left: 80px;
-    margin-right: 80px;
+    text-align: left;
+    margin-left: 45vh;
+    margin-right: 200px;
```

Comment:
having margin-left: 45vh means that when the screen size is smaller, a lot of the space will be empty.
having a fixed pixel size for margin-right also hurts the responsive design. 
Instead of doing margin on "p", try putting padding inside the whole div, try something like "padding-right: 10%", "padding-left: 10%"